src/js/tooltip.js

```diff
@@ -305,13 +325,13 @@ class Tooltip {
 
 		switch (this.tooltipPosition) {
 			case 'above':
-				rect.top = this.target.top - (height + Tooltip.arrowDepth);
-				rect.left = this._getLeftFor('middle');
+				rect.top = this.target.top - height - Tooltip.arrowDepth;
+				rect.left = this._getLeftFor("middle");
```

Comment:
I'm being my pedantic self, but why the double quote change? Everywhere else in the file still seems to have single quotes.